src/ocean/text/arguments/TimeIntervalArgs.d

```diff
@@ -142,48 +182,33 @@ public TimestampInterval processTimeIntervalArgs ( Arguments args )
     enforce(num_args >= 1 && num_args <= 2, "Not enough arguments provided");
     bool include_end_date = !args["time-interval-exclude"].set;
 
-    long begin, end;
```

Comment:
Should the docs for this method be updated as the functionality has changed?

Reply:
I think it's better to update the release notes